pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -14,8 +14,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "openmvg";
     repo = "openmvg";
-    rev = "v${version}";
-    sha256 = "sha256-6F/xUgZpqY+v6CpwTBhIXI4JdT8HVB0P5JzOL66AVd8=";
+    rev = "e1bbfe801986cd7171f36443a1573b0f69f3702d";
```

Comment:
Change version string to `unstable-YYYY-MM-DD` (the date of the rev you are using here).